rdfunit-core/src/main/java/org/aksw/rdfunit/tests/executors/monitors/SimpleTestExecutorMonitor.java

```diff
@@ -206,6 +206,7 @@ public void testingFinished() {
                 .setSchemata(schemata)
                 .setTestCaseUris(testCaseUris)
                 .setResults(results)
+                .setDefaultExecutionType()
```

Comment:
I already set the execution type before, I fixed the bug here yesterday https://github.com/AKSW/RDFUnit/commit/a5bd2f1b6135e41e567a7fbe4e27c128cb17fdab but forgot to push :)
but the test are good